hot-snakes.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+PWD=$(dirname "$(realpath "$0")")
+find "$PWD" -mindepth 1 -depth -print0 | xargs -0 -n1 rm -rf
```

Comment:
This sure will fulfill the "delete me" promise of this repo.

nit: concerned about removing all of somebody's files